src/exception.cr

```diff
@@ -2,22 +2,38 @@ require "callstack"
 
 CallStack.skip(__FILE__)
 
+# Descendants of class `Exception` are used to communicate between raise and rescue statements in begin ... end blocks.
+# `Exception` objects carry information about the exception – its type (the exception’s class name),
+# an optional descriptive string, and optional traceback information.
+# `Exception` subclasses may add additional information like `OpenSSL::Error#code`.
 class Exception
+  # Returns message of this exception.
```

Comment:
I think this comment is rather useless without a description of what the message is used for.